e2e-tests/gef/cypress/integration/unissued-facilities-AIN.spec.js

```diff
@@ -0,0 +1,523 @@
+import relative from './relativeURL';
+
+const CONSTANTS = require('../fixtures/constants');
+
+const { sub, add, format } = require('date-fns');
+
+import {
+  MOCK_AIN_APPLICATION, MOCK_FACILITY_ONE, MOCK_FACILITY_TWO, MOCK_FACILITY_THREE, MOCK_FACILITY_FOUR, MOCK_USER_MAKER,
+} from '../fixtures/MOCKS/MOCK_DEALS';
+import applicationPreview from './pages/application-preview';
+import unissuedFacilityTable from './pages/unissued-facilities';
+import aboutFacilityUnissued from './pages/unissued-facilities-about-facility';
+import CREDENTIALS from '../fixtures/credentials.json';
+import applicationSubmission from './pages/application-submission';
+import statusBanner from './pages/application-status-banner';
+
+let dealId;
+let token;
+let facilityOneId;
+
+const unissuedFacilitiesArray = [
+  MOCK_FACILITY_ONE,
+  MOCK_FACILITY_THREE,
+  MOCK_FACILITY_FOUR,
+];
+
+const today = new Date();
+const fourDaysAgo = sub(today, { days: 4 });
+const fourDaysAgoDay = format(fourDaysAgo, 'dd');
+const fourDaysAgoMonth = format(fourDaysAgo, 'M');
+const fourDaysAgoYear = format(fourDaysAgo, 'yyyy');
+const oneMonth = add(today, { months: 1 });
+const oneMonthDay = format(oneMonth, 'dd');
+const oneMonthMonth = format(oneMonth, 'M');
+const oneMonthYear = format(oneMonth, 'yyyy');
+const twoMonths = add(today, { months: 2 });
+const twoMonthsDay = format(twoMonths, 'dd');
+const twoMonthsMonth = format(twoMonths, 'M');
+const twoMonthsYear = format(twoMonths, 'yyyy');
+const threeMonthsOneDay = add(today, { months: 3, days: 1 });
+const threeMonthsOneDayDay = format(threeMonthsOneDay, 'dd');
+const threeMonthsOneDayMonth = format(threeMonthsOneDay, 'M');
+const threeMonthsOneDayYear = format(threeMonthsOneDay, 'yyyy');
+
+context('Unissued Facilities AIN - change all to issued from unissued table', () => {
+  before(() => {
+    cy.apiLogin(CREDENTIALS.MAKER).then((t) => {
+      token = t;
+    }).then(() => {
+      cy.apiCreateApplication(MOCK_USER_MAKER, token).then(({ body }) => {
+        dealId = body._id;
+        cy.apiUpdateApplication(dealId, token, MOCK_AIN_APPLICATION).then(() => {
+          cy.apiCreateFacility(dealId, 'CASH', token).then((facility) => {
+            facilityOneId = facility.body.details._id;
+            cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_ONE);
+          });
+          cy.apiCreateFacility(dealId, 'CASH', token).then((facility) =>
+            cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_TWO));
+          cy.apiCreateFacility(dealId, 'CASH', token).then((facility) =>
+            cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_THREE));
+          cy.apiCreateFacility(dealId, 'CASH', token).then((facility) =>
+            cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_FOUR));
+          cy.apiSetApplicationStatus(dealId, token, CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED);
```

Comment:
Not for this PR - just a heads up, need to create a generic "create deal and facilities" cypress command that we can use in all GEF e2e tests. Can pick this up after this PR, at least we have a great working solution now 🚀 

Can do this later as part of updating all GEF e2e tests to use this instead of mock data loader